### ISSUE TITLE
[Mechs] Preview window does not pan correctly

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#5816]
### Short Description of the issue:
The Entity Preview window does not pan when you click and drag it, so I'm unable to get a good look at the limbs of this mech

https://imgur.com/a/uL3vgnl

### Steps to reproduce the issue:
- Load the attached bp
- Enter the preview window and try to drag it around

### Environment
- Starship EVO version: Mech Preview
- Graphics Card: RX6700XRT

### Additional information:
[Dwarf Spider Droid.zip](https://github.com/user-attachments/files/15905664/Dwarf.Spider.Droid.zip)

**Comment by ProPeach** [comment]
@tsunamayo you're right I did mean rotate, not pan. It's odd as the preview in the Entiy Menu showed a better view of the mech, as well as allowing more rotation. The Mech Computer view was basically frozen in one orientation, you see in the video it doesn't move at all really only snapping a few degrees to one side at one point

**Comment by drimmel247** [comment]
@tsunamayo The spin is inconsistent, sometimes it lets you click-hold(slow rotate) to spin and sometime it will just do a 180 spin during that click hold where you only wanted like 45 degrees or anything less than 180.  

Side note: any large modelled mechs are only visible from the waist down, can we get a zoom feature for zooming in and zooming out of the motion? this will be helpful when starting to work on detailed motions like hands.

**Comment by ProPeach** [comment]
Here's another mech with both a strange camera orbit location (above and behind the mech) as well as the inconsistent and jumpy rotation 
https://imgur.com/a/LGLvyE2

[TIEm Mech Heavy Legs Walkin'.zip](https://github.com/user-attachments/files/16021310/TIEm.Mech.Heavy.Legs.Walkin.zip)

**Comment by ProPeach** [comment]
Here's another mech with both a strange camera orbit location (above and behind the mech) as well as the inconsistent and jumpy rotation
https://imgur.com/a/LGLvyE2

[TIEm Mech Heavy Legs Walkin'.zip](https://github.com/user-attachments/files/16021310/TIEm.Mech.Heavy.Legs.Walkin.zip)
